backend/python_init_script/main.py

```diff
@@ -32,33 +32,34 @@ def load_user_data(file: str) -> [UserData]:
     :return: iterable of the loaded UserData objects
     :rtype: [UserData]
     """
+
+    def _replace_nan(val) -> str:
+        # if cell has no value in xlsx/csv it will result in a nan value
+        return "" if str(val) == "nan" else str(val)
+
+    def _get_roles(row):
+        # split roles by ',' if there are more
```

Comment:
I'm sorry to comment again. I'm a bit reluctant with the comment because it seems superfluous because the code explains itself already.